pkg/apis/core/validation/validation_test.go

```diff
@@ -10288,12 +10288,12 @@ func TestValidateServiceCreate(t *testing.T) {
 			numErrs: 0,
 		},
 		{
-			name: "invalid load balancer with mix protocol",
+			name: "load balancer with mix protocol",
```

Comment:
non-blocking comment, there should be a test here that accounts for the feature gate

Reply:
The feature gate is checked in a different function than `ValidateServiceCreate`. The feature gate is checked in `ValidateConditionalService` and for that we have the unit tests.